File: Questions/Python/binbs.py
n=int(input())
str=input()
fs=''
i=0
count=0
while i<n:
    try:
        if str[i]=='0' and str[i+1]=='1' and str[i+2]=='0':
            fs+='011'
            i+=3
            count+=1
            continue
    except:
        fs+=str[i]        
    i+=1
print(count)

File: Questions/cpp/binbs.cpp
#include<iostream>
using namespace std;
int main()
{
    long long int n;
    cin>>n;
    string str;
    cin>>str;
    long long int count=0;
    for(int i=0;i<n-2;i++)
    {
        if (str[i]=='0' && str[i+1]=='1' && str[i+2]=='0')
        {
            str[i+2]=1;
            count++;
        }

    }
    cout<<count<<endl;
}
